src/antsibull/cli/antsibull_build.py

```diff
@@ -131,6 +131,16 @@ def _normalize_new_release_options(args: argparse.Namespace) -> None:
             )
 
 
+def _check_release_build_directories(args: argparse.Namespace) -> None:
+    if args.command in ('single', 'multiple', 'rebuild-single'):
+        if not os.path.isdir(args.sdist_dir):
+            raise InvalidArgumentError(f'{args.sdist_dir} must be an existing directory')
+
+    if args.command in ('rebuild-single', ):
+        if args.sdist_src_dir is not None and os.path.exists(args.sdist_src_dir):
+            raise InvalidArgumentError(f'{args.sdist_src_dir} must not exist')
```

Comment:
What do think about allowing this to work as long as the existing directory is empty? Doing so will need a bit more code, but this behavior seems more sensible to me. You'd also have to pass `dirs_exist_ok=True` (Python 3.8 and above only) to `shutil.copytree` above or I guess remove the directory first.

Reply:
Since antsibull supports Python 3.6.1+, using `dirs_exist_ok=True` is not possible. I don't think it's worth to invest this much extra work in a functionality that's only for debugging (I was even considering not making it show up in `--help`).

Reply:
> Since antsibull supports Python 3.6.1+

That might be worth reconsidering. The latest versions of ansible-core don't work at all with Python 3.6. Anyways, I've marked this as approved.

Reply:
Yeah, at some point we should drop support for a lot of no longer needed stuff. Right now there's also support to build Ansible 2.10 still included, and several other EOL Ansible versions.